libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -258,6 +258,9 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 // constructor
 AP_GPS::AP_GPS()
 {
+    static_assert((sizeof(_initialisation_blob) * 9) < (4800 * GPS_BAUD_TIME_MS * 1e-3),
```

Comment:
I knew I was crazy :smile: Of course the baud is in bits so you have to convert the size in bytes to bits. I'll bang my head a bit more :smiley: 

I agree with you that using a define here is too much. But maybe changing it to `* (CHAR_BIT + 1)` would make it more explicit (and portable to any crazy environment where a byte isn't 8 bits)? You'll need to include `<climits>` for that.

Anyway, I think that whatever you decide is fine.